src/oss/javascript/integrations/chat/anthropic.mdx

```diff
@@ -990,7 +990,7 @@ Learn more about tool calling [here](https://js.langchain.com/docs/how_to/tool_c
 
 ### Using with text splitters
 
-Anthropic also lets you specify your own splits using [custom document](https://docs.anthropic.com/en/docs/build-with-claude/citations#custom-content-documents) types. LangChain [text splitters](/oss/concepts/text_splitters/) can be used to generate meaningful splits for this purpose. See the below example, where we split the LangChain.js README (a markdown document) and pass it to Claude as context:
+Anthropic also lets you specify your own splits using [custom document](https://docs.anthropic.com/en/docs/build-with-claude/citations#custom-content-documents) types. LangChain text splitters can be used to generate meaningful splits for this purpose. See the below example, where we split the LangChain.js README (a markdown document) and pass it to Claude as context:
```

Comment:
we have text splitters: https://docs.langchain.com/oss/python/integrations/splitters

Reply:
we dont have it for js